src/kirby/kirby.common.ts

```diff
@@ -1,48 +1,49 @@
-import { GroupByPipe } from './components/list/pipes/group-by.pipe';
 import { AvatarComponent } from './components/avatar/avatar.component';
+import { BadgeComponent } from './components/badge/badge.component';
```

Comment:
What happened here? a try on sorting the imports?

Reply:
Yes, sorting was set up in my Vscode. I think it is okay here to have this sorted :)